scripts/lock.sh

```diff
@@ -2,8 +2,12 @@
 # file: lock.sh
 # description: Lock dependencies and export requirements.
 
+echo "THE FILE WAS GENERATED BY POETRY, DO NOT EDIT!\n\n" > src/requirements.txt
+echo "THE FILE WAS GENERATED BY POETRY, DO NOT EDIT!\n\n" > src/requirements-dev.txt
```

Comment:
Hey. I'm not sure what should I expect, but this doesn't seem to work for me with pip 20.0.2 on Python 3.8. It prints:
```
ERROR: Invalid requirement: 'THE FILE WAS GENERATED BY POETRY, DO NOT EDIT!' (from line 1 of src/requirements.txt)
```

I suppose this line should be commented like
```sh
echo "# THE FILE WAS GENERATED BY POETRY, DO NOT EDIT!\n\n" > src/requirements.txt
echo "# THE FILE WAS GENERATED BY POETRY, DO NOT EDIT!\n\n" > src/requirements-dev.txt
```
instead?

Reply:
Hi @upsuper,

That's a very good point. Those lines should indeed be comments. 

Could you possibly create a PR for this fix? If not, I'll try to find some time this weekend to fix this.